update-server-repo-setup/generate-repos-config

```diff
@@ -144,42 +146,48 @@ class RepoConfigGenerator:
         config_products = self.make_products_list()
         scc_products = self.get_scc_products()
         
-        result = {}
-        for triplet, product_settings in config_products.items():
-            exclusions = product_settings.get('exclusions')
-            scc_product = scc_products.get(triplet)
-            if not scc_product:
-                msg = 'WARNING: Configured product "%s" not provided by SCC. '
-                msg += 'Skipping product.'
-                print(msg % triplet)
-                continue
-            
-            for repo in scc_product:
-                fs_location = repo['url'].split('suse.com')[-1]
-                repo_name = repo['name']
-                skip = False
-                if (repo['enabled']):
-                    if self._add_repo(repo_name, exclusions):
-                        if self.verbose:
-                            print('Adding mandatory repo "%s"' % repo_name)
-                        result[repo['id']] = {
-                            'description': repo['description'],
-                            'framework': product_settings['framework'],
-                            'location': fs_location
-                        }
-                else:
-                    additional_repos = product_settings['additional_repos']
-                    for additional_repo in additional_repos:
+        result = []
```

Comment:
@ikapelyukhin I don't think I understand what you have in mind. The `del enabled_repos[id]` in the monitoring has been fixed.

The whole thing was a dict such that a config of `"arches": [ "x86_64:azure", "x86_64:ec2" ],` got lost.The only way to not loose multiple definitions is to have a list of some sort.

Reply:
I meant having the frameworks array nested inside the repo element (`{ "123": { "frameworks": ["ec2", "azure"], ... }, ... }`) rather then having repos repeat -- but never mind that since it was already addressed in the monitoring script.